# 
# Submitted-By: Benedikt Spranger, 2005-09-26
# Committed-By: Robert Schwebel
#
# Error: 
#
# 
#
# Description: 
#
# GCC 4.x declares non-static declaration and static definition and vice 
# versa as error.
#
# State:
# 
# unknown
#

diff -ruN grub-0.97.orig/netboot/main.c grub-0.97/netboot/main.c
--- grub-0.97.orig/netboot/main.c	2004-05-21 00:19:33.000000000 +0200
+++ grub-0.97/netboot/main.c	2005-09-15 17:33:34.383904479 +0200
@@ -54,9 +54,9 @@
 
 static int vendorext_isvalid;
 static unsigned long netmask;
-static struct bootpd_t bootp_data;
+struct bootpd_t bootp_data;
 static unsigned long xid;
-static unsigned char *end_of_rfc1533 = NULL;
+unsigned char *end_of_rfc1533 = NULL;
 
 #ifndef	NO_DHCP_SUPPORT
 #endif /* NO_DHCP_SUPPORT */
